webpack.config.js

```diff
@@ -1,7 +1,12 @@
 var fs = require('fs');
 var path = require('path');
 
+var webpack = require('webpack');
+
 const debugBuild = process.env.NODE_ENV === 'development';
+var definePlugin = new webpack.DefinePlugin({
+    __DEBUG__: debugBuild,
```

Comment:
Isn't this already done in the minify-replace Babel plugin below? Should it replace Babel's minify-replace?

Reply:
yes, thanks, it's a duplicate, I remove it